HelpSource/Classes/List.schelp

```diff
@@ -6,16 +6,36 @@ categories::Collections>Ordered
 DESCRIPTION::
 List is a subclass of SequenceableCollection with unlimited growth in size. Although not a subclass of link::Classes/Array:: or its superclass link::Classes/ArrayedCollection:: it uses an Array in its implementation and is in many cases interchangeable with one. (List implements many of the same methods as Array.)
 
-Arrays have a fixed maximum size. If you add beyond that size a new Array is created and returned, but you must use an assignment statement or the new array will be lost. (See the link::Classes/Array:: helpfile.) List has no size limitation and is thus more flexible, but has slightly more overhead.
+note::
+Size limitations of arrays and flexibility of lists in SuperCollider
+
+Arrays have a fixed maximum size. If you add beyond that size, a new link::Classes/Array:: is created and returned, but you must use an assignment statement or the new array will be lost. (See the link::Classes/Array#description#Description:: section in link::Classes/Array::. documentation). The idiomatic way to add new item in link::Classes/Array:: is:
+
+code::
+x = x.add(i);
+::
+
+Example:
+
 code::
 (
```

Comment:
Since we want to make the examples accessible, it is better to emphasise the readability of the code:

```supercollider
(
x = Array.new(3); y = nil;
"Make an empty array of size 3: \nx:% \ny: %".format(x, y).postln;

5.do { |i| y = x.add(i) };

"After adding 5 items to each: \nx:%\ny: %".format(x, y).postln;
"Accessing:\nx.at(4): \nx:%\ny: %".format(x.at(4), y[4]).postln;"";
)
```

Reply:
Thanks. done in
https://github.com/supercollider/supercollider/pull/6536/commits/87bea3eeaeb0ab3b12fec0958b35647a1249ff34!

@mtmccrea:
I added comments in the code example. Please leave further comments on the new commit! Thanks!